### DISCUSSION TITLE
How to access request object in server actions?

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [vercel/next.js#65326] [Help]
### Summary

Hi, I want to be able to access the request object in server actions, the same way that we do in regular requests and api requests.

**Comment by Nadeera3784** [comment]
Hi @aprilmintacpineda ,
This might help you  https://github.com/vercel/next.js/discussions/57949

**Reply by aprilmintacpineda** [reply]
That actually kinda did... Thanks!
